### PR TITLE
Added models. Added associations, validations and specs to them.

### PR DESCRIPTION
**Pull request by Zawgor24**

### CODE REVIEW

File: app/admin/dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  content title: proc { I18n.t('active_admin.dashboard') } do
+    div class: 'blank_slate_container', id: 'dashboard_default_message' do
+      span class: 'blank_slate' do
+        span I18n.t('active_admin.dashboard_welcome.welcome')
+        small I18n.t('active_admin.dashboard_welcome.call_to_action')
+      end
+    end
+
+    # Here is an example of a simple dashboard with columns and panels.
```
Comment (IlyaUmanets) [inline app/admin/dashboard.rb]: Remove comments

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -2,3 +2,5 @@
  *= require_tree .
  *= require_self
  */
+@import 'bootstrap';
+@import 'toastr';
```
Comment (IlyaUmanets) [inline app/assets/stylesheets/application.scss]: Add new line

File: app/helpers/categories_helper.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
```
Comment (IlyaUmanets) [inline app/helpers/categories_helper.rb]: Remove empty helpers.

File: app/models/category.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Category < ApplicationRecord
+  has_many :posts, dependent: :nullify
+  has_many :articles, dependent: :nullify
```
Comment (IlyaUmanets) [inline app/models/category.rb]: Sort association by alphabet

File: app/models/comment.rb [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :commentable, polymorphic: true
```
Comment (IlyaUmanets) [inline app/models/comment.rb]: Sort by alpha

File: app/models/post.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user
+  belongs_to :category, required: false
```
Comment (IlyaUmanets) [inline app/models/post.rb]: Same

File: spec/factories/articles.rb [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :article do
+    title 'MyString'
```
Comment (IlyaUmanets) [inline spec/factories/articles.rb]: `MyString` is not best value.

File: spec/factories/posts.rb [hunk 8]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :post do
+    title { Faker::Hacker.abbreviation }
+    body { Faker::Hacker.say_something_smart }
+    avatar { Faker::Avatar.image }
+    category nil
```
Comment (IlyaUmanets) [inline spec/factories/posts.rb]: You don't need to specify `category nil`. You will get same result if you remove this line

File: spec/helpers/categories_helper_spec.rb [hunk 9]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
```
Comment (IlyaUmanets) [inline spec/helpers/categories_helper_spec.rb]: Remove helper specs

File: spec/models/article_spec.rb [hunk 10]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  describe 'Associations' do
+    it { is_expected.to belong_to(:user) }
```
Comment (IlyaUmanets) [inline spec/models/article_spec.rb]: Sort by alpha

File: spec/models/article_spec.rb [hunk 11]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  describe 'Associations' do
```
Comment (IlyaUmanets) [inline spec/models/article_spec.rb]: lowercase. just `associations`

**Review by IlyaUmanets — CHANGES_REQUESTED** [review]

File: app/admin/dashboard.rb (hunk 1 above)
Comment (Zawgor24) [reply app/admin/dashboard.rb]: Removed all commented lines.

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (Zawgor24) [reply app/assets/stylesheets/application.scss]: Added last line.

File: app/helpers/categories_helper.rb (hunk 3 above)
Comment (Zawgor24) [reply app/helpers/categories_helper.rb]: Removed all helpers.

File: app/models/category.rb (hunk 4 above)
Comment (Zawgor24) [reply app/models/category.rb]: Sorted.

File: app/models/comment.rb (hunk 5 above)
Comment (Zawgor24) [reply app/models/comment.rb]: Done.

File: app/models/post.rb (hunk 6 above)
Comment (Zawgor24) [reply app/models/post.rb]: Done.

File: spec/factories/articles.rb (hunk 7 above)
Comment (Zawgor24) [reply spec/factories/articles.rb]: Got it, changed to some Faker method.

File: spec/factories/posts.rb (hunk 8 above)
Comment (Zawgor24) [reply spec/factories/posts.rb]: OK, removed.

File: spec/helpers/categories_helper_spec.rb (hunk 9 above)
Comment (Zawgor24) [reply spec/helpers/categories_helper_spec.rb]: Removed.

File: spec/models/article_spec.rb (hunk 11 above)
Comment (Zawgor24) [reply spec/models/article_spec.rb]: Got it, done.

File: spec/models/article_spec.rb (hunk 10 above)
Comment (Zawgor24) [reply spec/models/article_spec.rb]: Done.

**Review by IlyaUmanets — APPROVED** [review]

**Pull request merged**